### ISSUE TITLE
[Northrend] [The Drakkensryd - quest worked but now Im stuck and can't do anything.]

### ISSUE THREAD

**Issue by Rocknblock** [dalaranwow/dalaran-wow#5362]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
On this quest, I was hoping around from drake to drake and killing the npc's on the drake just fine. Then I completed the quest. After that I used the quest item to go to the column ornament so I could get my feet back on the ground. I was pulled over to it just fine but now I'm stuck and can't move. I tried grappling over to a different column ornament but it wouldn't let me. I typed /reload thinking that might help but I'm still stuck. I'm not able to even use the hearthstone. :(  So I'm sitting here not able to do a thing. Can someone please help me? 
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Jump to drakes, fight red npcs.
2. Then I jumped on the closest drake to the pillar. it was pretty much even with the pillars floor.
3. Grapple over to the Column Ornament(A glowing crystal ball that is attached to one of the smaller pillars.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rocknblock** [comment]
I just exited the game and then logged back in. That seemed to get me unstuck. Seems like that problem would be a good candidate to get fixed though when you all have the time to. :)

**Comment by galgnu** [comment]
Had the excact same experience when I did the quest.

**Comment by Rushor** [comment]
moved to https://github.com/dalaranwow/dalaran-wow/issues/3904
